README.md

```diff
@@ -46,6 +46,16 @@ The runtime has a built-in command to determine if your host system is capable o
 $ cc-runtime cc-check
 ```
 
+Note:
+
+If you run the command above as the `root` user, further checks will be
+performed (such as checking if another incompatible hypervisor is
```

Comment:
Lines 52-53 suggested rewrite: performed (e.g. check if another incompatible hypervisor is running):